paraatm/io/weather_cube_ET.py

```diff
@@ -0,0 +1,361 @@
+#!/home/anaconda3 python
+# -*- coding: utf-8 -*-
+"""
+NASA NextGen NAS ULI Information Fusion
+
+@organization: Arizona State University
+@author: Yutian Pang
+@date: 2019-02-19
+@last updated: 2020-06-01
+
+This script is used to read and process the EchoTop (ET) convective weather products from
+the Corridor Integrated Weather System (CIWS). The data can be downloaded from NASA
+Sherlock Data Warehouse. url: https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20180004248.pdf
+
+The script takes raw ET convective weather files and the trajectory file as inputs. The outputs are
+the processed weather feature cube around each of the coordinates in trajectory file, as well as the
+value of the ET right at each coordinate.
+
+"""
+
+import os
+import time
+import math
+import datetime
+import numpy as np
+import pandas as pd
+from netCDF4 import Dataset
+
+
+class weather_cube_generator(object):

```

Comment:
Please use "CapWords" style for class names.  This is discussed in the "Style" section of the [Contributing Guide](https://github.com/ymlasu/para-atm/wiki/Contributing).

Reply:
Thanks. I will modify the class name.